src/Go.CodeAnalysis/Parser/ImportNode.cs

```diff
@@ -6,29 +6,101 @@
     using Go.CodeAnalysis.Lex;
     using Go.CodeAnalysis.Text;
 
+    /// <summary>
+    /// Represents a single import line or import block.
+    /// </summary>
     public sealed class ImportNode : ParseNode
     {
-        public ImportNode(SnapshotSegment extent, SnapshotSegment importExtent) : base(extent, ImmutableArray<ParseNode>.Empty)
+        public ImportNode(
+            SnapshotSegment extent,
+            ImmutableArray<ImportDeclarationNode> importDeclarations)
+            : base(extent, ImmutableArray<ParseNode>.Empty)
         {
-            this.ImportExtent = importExtent;
+            this.ImportExtent = extent;
+            this.ImportDeclarations = importDeclarations;
         }
 
         public SnapshotSegment ImportExtent { get; }
 
+        public ImmutableArray<ImportDeclarationNode> ImportDeclarations { get; }
+
         public static bool TryParse(Lexer lexer, IList<Error> errors, out ImportNode parseNode)
         {
             var start = lexer.CurrentLexeme.Extent.Start;
+            parseNode = null;
+
+            // Parse "import" keyword.
+            if (!lexer.IsCorrectLexemeKeywordOrReportError(Keywords.Import, errors)
+                || !lexer.TryAdvanceLexemeOrReportError(errors))
+            {
+                errors.Add(new Error(lexer.CurrentLexeme.Extent, "Import keyword expected."));
+                return false;
+            }
+
+            // Single line import (string immediately following import keyword).
+            if (!lexer.IsCorrectLexemeOperatorOrReportError('(', errors))
```

Comment:
I think this will cause an error for the non-parenthesized form. Maybe this should be `IsCorrectLexmeOperator()`?

Reply:
Added methods without error reporting.